src/promptflow/promptflow/_sdk/_pf_client.py

```diff
@@ -157,16 +157,19 @@ def run(
             )
         if not flow:
             raise ValueError("'flow' is required to create a run.")
-        if not os.path.exists(flow) and not is_python_flex_flow_entry(entry=flow):
-            # check if it's eager flow's entry
+        if callable(flow):
+            logger.debug(f"flow entry {flow} is a callable.")
+        elif is_python_flex_flow_entry(entry=flow):
+            logger.debug(f"flow entry {flow} is a python flex flow.")
+        elif os.path.exists(flow):
+            logger.debug(f"flow entry {flow} is a local path.")
+        else:
             raise UserErrorException(f"Flow path {flow} does not exist and it's not a valid entry point.")
         if data and not os.path.exists(data):
             raise FileNotFoundError(f"data path {data} does not exist")
         if not run and not data:
             raise ValueError("at least one of data or run must be provided")
-        if code and not os.path.exists(code):
-            raise FileNotFoundError(f"code path {code} does not exist")
-        code = Path(code) if code else Path(os.getcwd())
+
```

Comment:
Why this got removed?

Reply:
offline synced, moved inside `generate_yaml_entry` since it's related to the logic